.github/workflows/stale.yml

```
@@ -15,7 +15,7 @@ jobs:
         stale-issue-message: "Hello 👋, This issue has been inactive for over 9 months. To help maintain a clean and focused backlog, we'll be marking this issue as stale and will close the issue if we detect no activity in the next 7 days. Thank you for your contribution and understanding! 🙏"
         close-issue-message: "Hello 👋, This issue has been inactive for over 9 months and hasn't received any updates since it was marked as stale. We'll be closing this issue for now, but if you believe this issue is still relevant, please feel free to reopen it. Thank you for your contribution and understanding! 🙏"
         stale-issue-label: "stale"
-        exempt-issue-labels: "needs discussion"  # Comma-separated list of labels.
+        exempt-issue-labels: "needs discussion", "untriaged"  # Comma-separated list of labels.
```

actually, this has to be
```suggestion
        exempt-issue-labels: "needs discussion,untriaged"  # Comma-separated list of labels.
```